plugins/splunk-on-call/src/index.ts

```diff
@@ -14,14 +14,12 @@
  * limitations under the License.
  */
 export {
+  SplunkOnCallCard,
   splunkOnCallPlugin,
   splunkOnCallPlugin as plugin,
   SplunkOnCallPage,
 } from './plugin';
-export {
-  isPluginApplicableToEntity,
-  SplunkOnCallCard,
-} from './components/SplunkOnCallCard';
+export { isSplunkOnCallAvailable } from './components/EntitySplunkOnCallCard';
```

Comment:
Does this also need to include `EntitySplunkOnCallCard`?

Reply:
I think it shouldn't; the one exported from the plugin above should be renamed to `EntitySplunkOnCallCard` instead.